Libraries/Network/XMLHttpRequestBase.js

```diff
@@ -139,6 +143,15 @@ class XMLHttpRequestBase {
     }
   }
 
+  _stringToArrayBuffer(str: string): ArrayBuffer {
+    var buffer = new ArrayBuffer(str.length);
+    var bufferView = new Uint8Array(buffer);
```

Comment:
Looks like iOS7 market share is less than 6%  https://developer.apple.com/support/app-store/. Do you recommend enabling this feature with a flag by checking the iOS version? @javache